package.json

```diff
@@ -15,8 +15,9 @@
     "start": "node server.js",
     "start:dev": "hof-build watch",
     "dev": "hof-build watch --env",
-    "test": "yarn run test:lint",
+    "test": "yarn run test:lint && yarn run test:unit",
     "test:lint": "eslint .",
+    "test:unit": "NODE_ENV=test nyc _mocha \"test/_unit/**/*.spec.js\"",
```

Comment:
I would prefer to also just have a unit test script on it's own but not important for now.

Reply:
not sure what you mean by unit test script on it's own?

Reply:
This I believe runs the unit tests and also the coverage.  When I'm fixing unit tests, I don't really want the coverage malarky in my way so I just want a script to run the unit tests 😄

Reply:
don't worry, I'll add it myself